### PR TITLE
Recent AUV calibrations

### PR DESCRIPTION
**Pull request by cdobs**

### CODE REVIEW

File: calibration/PARADN/CGINS-PARADN-50170__20220622.csv [hunk 1]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+50170,CC_dark_offset,9.4,in mV; source_file PARAD-N_QSP2150A_SN_50170_Calibration_Certificate_2022-06-22.pdf
+50170,CC_scale_wet,6.977,in Volts/(uE/cm^2sec)
```
Comment (rtravis-whoi) [inline calibration/PARADN/CGINS-PARADN-50170__20220622.csv:3]: cal date in filename should be 20220610

File: calibration/PARADN/CGINS-PARADN-50372__20220713.csv [hunk 2]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+50372,CC_dark_offset,11,in mV; source_file PARAD-N_QSP2150A_SN_50372_Calibration_Certificate_2022-07-13.pdf
+50372,CC_scale_wet,6.633,in Volts/(uE/cm^2sec)
```
Comment (rtravis-whoi) [inline calibration/PARADN/CGINS-PARADN-50372__20220713.csv:3]: good

File: calibration/NUTNRN/CGINS-NUTNRN-01121__20221104.csv [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
...273.42, 274.24, 275.06, 275.88, 276.7, 277.52, 278.34, 279.16, 279.98, 280.8, 281.62, 282.44, 283.26, 284.08, 284.9, 285.72, 286.54, 287.36, 288.18, 289, 289.82, 290.64, 291.46, 292.28, 293.09, 293.91, 294.73, 295.55, 296.37, 297.19, 298.01, 298.83, 299.64, 300.46, 301.28, 302.1, 302.92, 303.74, 304.55, 305.37, 306.19, 307.01, 307.83, 308.64, 309.46, 310.28, 311.09, 311.91, 312.73, 313.55, 314.36, 315.18, 316, 316.81, 317.63, 318.44, 319.26, 320.08, 320.89, 321.71, 322.52, 323.34, 324.15, 324.97, 325.78, 326.6, 327.41, 328.23, 329.04, 329.86, 330.67, 331.49, 332.3, 333.11, 333.93, 334.74, 335.55, 336.37, 337.18, 337.99, 338.8, 339.62, 340.43, 341.24, 342.05, 342.86, 343.68, 344.49, 345.3, 346.11, 346.92, 347.73, 348.54, 349.35, 350.16, 350.97, 351.78, 352.59, 353.4, 354.21, 355.02, 355.82, 356.63, 357.44, 358.25, 359.06, 359.86, 360.67, 361.48, 362.28, 363.09, 363.9, 364.7, 365.51, 366.31, 367.12, 367.92, 368.73, 369.53, 370.34, 371.14, 371.95, 372.75, 373.55, 374.36, 375.16, 375.96, 376.76, 377.57, 378.37, 379.17, 379.97, 380.77, 381.57, 382.37, 383.17, 383.97, 384.77, 385.57, 386.37, 387.17, 387.97, 388.77, 389.57, 390.36, 391.16, 391.96, 392.75, 393.55, 394.35, 395.14, 395.94]",
```
Comment (rtravis-whoi) [inline calibration/NUTNRN/CGINS-NUTNRN-01121__20221104.csv:8]: good

File: calibration/NUTNRN/CGINS-NUTNRN-01109__20221104.csv [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
...3, 273.43, 274.22, 275.01, 275.81, 276.6, 277.4, 278.19, 278.99, 279.78, 280.58, 281.37, 282.16, 282.96, 283.75, 284.55, 285.34, 286.14, 286.93, 287.72, 288.52, 289.31, 290.11, 290.9, 291.7, 292.49, 293.28, 294.08, 294.87, 295.67, 296.46, 297.25, 298.05, 298.84, 299.64, 300.43, 301.22, 302.02, 302.81, 303.61, 304.4, 305.19, 305.99, 306.78, 307.57, 308.37, 309.16, 309.95, 310.75, 311.54, 312.33, 313.13, 313.92, 314.71, 315.51, 316.3, 317.09, 317.88, 318.68, 319.47, 320.26, 321.05, 321.84, 322.64, 323.43, 324.22, 325.01, 325.8, 326.6, 327.39, 328.18, 328.97, 329.76, 330.55, 331.34, 332.13, 332.93, 333.72, 334.51, 335.3, 336.09, 336.88, 337.67, 338.46, 339.25, 340.04, 340.83, 341.62, 342.41, 343.19, 343.98, 344.77, 345.56, 346.35, 347.14, 347.93, 348.71, 349.5, 350.29, 351.08, 351.87, 352.65, 353.44, 354.23, 355.01, 355.8, 356.59, 357.37, 358.16, 358.95, 359.73, 360.52, 361.3, 362.09, 362.88, 363.66, 364.45, 365.23, 366.01, 366.8, 367.58, 368.37, 369.15, 369.93, 370.72, 371.5, 372.28, 373.07, 373.85, 374.63, 375.41, 376.2, 376.98, 377.76, 378.54, 379.32, 380.1, 380.88, 381.66, 382.44, 383.22, 384, 384.78, 385.56, 386.34, 387.12, 387.9, 388.68, 389.46, 390.23, 391.01, 391.79, 392.57]",
```
Comment (rtravis-whoi) [inline calibration/NUTNRN/CGINS-NUTNRN-01109__20221104.csv:8]: good

File: calibration/DOSTAN/CGINS-DOSTAN-01520__20220513.csv [hunk 5]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+1520,CC_calibration_date,1652400000000,sensor calibration date (milliseconds since 1970-01-01)
```
Comment (rtravis-whoi) [inline calibration/DOSTAN/CGINS-DOSTAN-01520__20220513.csv:2]: good

File: calibration/CTDAVN/CGINS-CTDAVN-00424__20220531.csv [hunk 6]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+424,CC_calibration_date,1653955200000,sensor calibration date (milliseconds since 1970-01-01)
```
Comment (snwhite) [inline calibration/CTDAVN/CGINS-CTDAVN-00424__20220531.csv:2]: Looks good.

File: calibration/CTDAVN/CGINS-CTDAVN-00425__20220531.csv [hunk 7]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+425,CC_calibration_date,1653955200000,sensor calibration date (milliseconds since 1970-01-01)
```
Comment (snwhite) [inline calibration/CTDAVN/CGINS-CTDAVN-00425__20220531.csv:2]: Looks good.

File: calibration/DOSTAN/CGINS-DOSTAN-01519__20220513.csv [hunk 8]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+1519,CC_calibration_date,1652400000000,sensor calibration date (milliseconds since 1970-01-01)
```
Comment (snwhite) [inline calibration/DOSTAN/CGINS-DOSTAN-01519__20220513.csv:2]: Looks good.

File: calibration/DOSTAN/CGINS-DOSTAN-01520__20220513.csv (hunk 5 above)
Comment (snwhite) [inline calibration/DOSTAN/CGINS-DOSTAN-01520__20220513.csv:2]: Looks good.

File: calibration/DOSTAN/CGINS-DOSTAN-01519__20220513.csv (hunk 8 above)
Comment (rtravis-whoi) [inline calibration/DOSTAN/CGINS-DOSTAN-01519__20220513.csv:2]: good

File: calibration/CTDAVN/CGINS-CTDAVN-00425__20220531.csv (hunk 7 above)
Comment (rtravis-whoi) [inline calibration/CTDAVN/CGINS-CTDAVN-00425__20220531.csv:2]: good

File: calibration/CTDAVN/CGINS-CTDAVN-00424__20220531.csv (hunk 6 above)
Comment (rtravis-whoi) [inline calibration/CTDAVN/CGINS-CTDAVN-00424__20220531.csv:2]: good

**Review by rtravis-whoi — APPROVED** [review]
cal date in filename for PARAD 50170 should be 20220610

**Review by rtravis-whoi — CHANGES_REQUESTED** [review]
cal date in filename should be 20220610

File: calibration/PARADN/CGINS-PARADN-50170__20220610.csv [hunk 9]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+50170,CC_dark_offset,9.4,in mV; source_file PARAD-N_QSP2150A_SN_50170_Calibration_Certificate_2022-06-10.pdf
+50170,CC_scale_wet,6.977,in Volts/(uE/cm^2sec)
```
Comment (rtravis-whoi) [inline calibration/PARADN/CGINS-PARADN-50170__20220610.csv:3]: update looks good!

File: calibration/NUTNRN/CGINS-NUTNRN-01109__20221104.csv (hunk 4 above)
Comment (snwhite) [inline calibration/NUTNRN/CGINS-NUTNRN-01109__20221104.csv:8]: Looks good.

**Review by rtravis-whoi — APPROVED** [review]

File: calibration/PARADN/CGINS-PARADN-50170__20220610.csv (hunk 9 above)
Comment (snwhite) [reply calibration/PARADN/CGINS-PARADN-50170__20220610.csv:3]: Looks good.

**Review by snwhite — APPROVED** [review]

**Pull request merged**